operations/kubernetes/Troubleshoot_Postgres_Database.md

```diff
@@ -175,8 +175,87 @@ postgres@keycloak-postgres-2:~$ patronictl list
 |      Cluster      |        Member       |     Host     |  Role  |  State  | TL | Lag in MB |
 +-------------------+---------------------+--------------+--------+---------+----+-----------+
 | keycloak-postgres | keycloak-postgres-0 | 10.42.10.22  |        | running | 47 |         0 |
-| keycloak-postgres | keycloak-postgres-1 | 10.40.11.191 | Leader | running | 47 |         0 |
-| keycloak-postgres | keycloak-postgres-2 | 10.40.11.190 |        | running | 47 |           |
+| keycloak-postgres | keycloak-postgres-1 | 10.40.11.191 | Leader | running | 47 |           |
+| keycloak-postgres | keycloak-postgres-2 | 10.40.11.190 |        | running | 47 |         0 |
++-------------------+---------------------+--------------+--------+---------+----+-----------+
+```
+
+If a cluster member is `stopped` after a successful reinitialization, check for pg_internal.init.* files that may need to be cleaned up. This can occur if the pgdata disk was full prior to the reinitialization, leaving truncated pg_internal.init.* files in the pgdata directory.
+
+```bash
+ncn-w001# kubectl exec keycloak-postgres-0 -c postgres -n services -it -- bash
+```
+
+Example output:
+
+```
+postgres@keycloak-postgres-2:~$ patronictl list
++-------------------+---------------------+--------------+--------+---------+----+-----------+
+|      Cluster      |        Member       |     Host     |  Role  |  State  | TL | Lag in MB |
++-------------------+---------------------+--------------+--------+---------+----+-----------+
+| keycloak-postgres | keycloak-postgres-0 | 10.42.10.22  |        | running | 47 |         0 |
+| keycloak-postgres | keycloak-postgres-1 | 10.40.11.191 | Leader | running | 47 |           |
+| keycloak-postgres | keycloak-postgres-2 | 10.40.11.190 |        | stopped |    |   unknown |
++-------------------+---------------------+--------------+--------+---------+----+-----------+
+```
+ 
+Exec into that pod that is `stopped` and check the most recent postgres log for any `invalid segment number 0` errors relating to pg_internal.init.* files.
+
+```bash
+ncn-w001# kubectl exec keycloak-postgres-2 -n services -it -- bash
+postgres@keycloak-postgres-2:~$ export LOG=`ls -t /home/postgres/pgdata/pgroot/pg_log/*.csv | head -1`
+```
+
+Example output:
+
+```
```

Comment:
this one is missing `bash`
Also, why is this example output? Looks like it's both input and output.
This applies throughout this file.

Reply:
I adjusted this one and only included the output.  These 'Example output 'style' was added by Nathan so I was just looking to follow this for the added steps where it made the most sense
https://github.com/Cray-HPE/docs-csm/commit/1dda90017b4b186a0aada19951e051f36d259f6a#diff-b584387e3d695e67c3ca09ebddd16dc7328a62b06fae402d42b58f1c9e67f4b5